src/Microsoft.AspNetCore.Mvc.Razor/RazorViewEngine.cs

```diff
@@ -46,60 +48,30 @@ public class RazorViewEngine : IRazorViewEngine
             IRazorPageFactoryProvider pageFactory,
             IRazorPageActivator pageActivator,
             HtmlEncoder htmlEncoder,
-            IOptions<RazorViewEngineOptions> optionsAccessor,
+            IOptions<RazorViewEngineOptions> options,
```

Comment:
+1 this is now more confusing because you're using the names `options` and `_options` for different types: `IOptions<RazorViewEngineOptions>` and `RazorViewEngineOptions`. Revert the parameter name change.